server/middleware/auth.js

```diff
@@ -1,10 +1,23 @@
+const REDIRECT_ROUTES = {
```

Comment:
I left it as '/todos' for reference since we will probably have routes like '/account' or '/historicalData'. Just thought this would be a future-proof solution, but I can hard code it for now, no problem

Reply:
I like the idea, and we might do something similar once we set up the routes on the React side. But I've gotta research with drey the best way to do it.

If we add it back it'll probably be at a higher level so the object can be referenced by _any_ of the route files.